lib/serviceClient.ts

```diff
@@ -259,6 +257,9 @@ export class ServiceClient {
                 queryParameterValue = encodeURIComponent(queryParameterValue);
               }
             }
+            if (queryParameter.collectionFormat !== QueryCollectionFormat.Multi) {
```

Comment:
What if we are not dealing with arrays at all? Shouldnt this be
```suggestion
            if (queryParameter.collectionFormat != undefined && queryParameter.collectionFormat !== QueryCollectionFormat.Multi) {
```

Reply:
I dont see the change for the above comment

Reply:
Might have been missed. Committing it now